azure/resource-containers/pf-management-group/functions/private/get-pfazmanagementgroup.ps1

```diff
@@ -0,0 +1,12 @@
+Function Get-PfAzManagementGroup{
+    Param (
```

Comment:
This is the private function now that will use the az module to get the management group from Azure